assets/javascripts/discourse/components/upcoming-events-calendar.js

```diff
@@ -5,6 +5,7 @@ import loadScript from "discourse/lib/load-script";
 import getURL from "discourse-common/lib/get-url";
 import { formatEventName } from "../helpers/format-event-name";
 import { isNotFullDayEvent } from "../lib/guess-best-date-format";
+import { _buildPopover, _destroyPopover } from "../lib/popover";
```

Comment:
Not a blocker but I'd leave the `_` prefix for private (non-exported) functions.